pipeline/extract/analytics.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from prefect import task
+
+from pipeline.types import AnalyticsEvent
+
+
+@task
+def extract_analytics(db) -> List[AnalyticsEvent]:
+    ref = db.reference(f"analytics/")
+    raw = ref.get()
+    list_of_analytics = []
+    for record in raw.values():
+        event = AnalyticsEvent(
```

Comment:
Can you also bring in the two new values that I added in #359?

Reply:
Added!